Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift

```diff
@@ -233,15 +234,16 @@ public final class HTTP2StreamMultiplexer: ChannelInboundHandler, ChannelOutboun
     ///         `createStreamChannel`.
     public init(mode: NIOHTTP2Handler.ParserMode,
                 channel: Channel,
-                targetWindowSize: Int = 65535,
+                targetWindowSize: Int32,
```

Comment:
This addition also needs a new function.

Reply:
Same here re: dropping this change.

Reply:
Will drop.

Reply:
I removed the default value to guard against coder error in the convenience, forcing use of the convenience init's arg, but doing so required fixing all the calls in tests.